airstory.php

```diff
@@ -16,6 +16,9 @@
 
 namespace Airstory;
 
+// Declare the canonical plugin version.
+define( 'AIRSTORY_VERSION', '0.1.0' );
```

Comment:
#37 has been opened and the milestone updated to make sure this happens for release.